config/v1/types_cluster_version.go

```diff
@@ -199,6 +199,15 @@ type ClusterVersionStatus struct {
 	// +listType=atomic
 	// +optional
 	ConditionalUpdates []ConditionalUpdate `json:"conditionalUpdates,omitempty"`
+
+	// relatedObjects is a list of objects that are "interesting" or related to this operator.
+	// `oc adm inspect` honors this field in any type to navigate and collect related data.
+	// Common uses are:
+	// 1. operator namespaces
+	// 2. operand namespaces
+	// +optional
+	// +openshift:enable:FeatureGate=CVORelatedObjects
+	RelatedObjects []ObjectReference `json:"relatedObjects,omitempty"`
```

Comment:
`+listType=map` and appropriate `listMapKey` missing

Reply:
> +listType=map and appropriate listMapKey missing

Will do.  Consideration for our future, will we fix the existing usages too?

Reply:
If they're in status, I would expect we are already doing the right thing and so should be fine to fix it

I also assume these would ratchet 🤔

Reply:
Oh very neat.  I think you can only make map keys on required fields and not all the fields are logically required.  I might go with atomic (single writer of this thing anyway)

Reply:
> I think you can only make map keys on required fields and not all the fields are logically required

Yes I agree, the required ones are `group`, `name` and `resource`, so potential overlap on `namespace`? 

If you don't have the listType enforcing the uniqueness, how about CEL?